src/status_im2/navigation/events.cljs

```diff
@@ -12,17 +14,12 @@
 
 (rf/defn navigate-to
   {:events [:navigate-to]}
-  [{:keys [db]} go-to-view-id screen-params]
-  (merge
-   {:db          (-> (assoc db :view-id go-to-view-id)
-                     (all-screens-params go-to-view-id screen-params))
-    :navigate-to go-to-view-id
-    :dispatch    [:hide-bottom-sheet]}
-   (when (#{:chat :community-overview} go-to-view-id)
-     {:dispatch-later
-      ;; 300 ms delay because, navigation is priority over shell card update
-      [{:dispatch [:shell/add-switcher-card go-to-view-id screen-params]
-        :ms       300}]})))
+  [{:keys [db] :as cofx} go-to-view-id screen-params]
+  (rf/merge
+   cofx
+   {:db         (all-screens-params db go-to-view-id screen-params)
+    :dispatch-n [[:hide-bottom-sheet]]}
+   (shell.events/shell-navigate-to go-to-view-id screen-params nil nil)))
```

Comment:
hmm, these changes are complicating what is already an overly complicated part of the code base. 
Is it possible/ Would it not be better to just create a separate event for shell navigations and just leave standard navigations as is.

i.e we have 
```
(rf/defn navigate-to
  {:events [:navigate-to]}
  ...
  ```
  
  ```
(rf/defn shell-navigate-to
  {:events [:shell-navigate-to]}
  ...
  (shell.events/shell-navigate-to go-to-view-id screen-params nil nil)))
  ```
etc..
WDYT? cc @flexsurfer, @ulisesmac @ilmotta

Reply:
There's a case to be made that the smaller the API surface the simpler, as long as the simpler model is still easy to use/call. What I mean is that having a single, canonical way to navigate to any screen by view ID is desirable, hence what this PR is doing kind of makes sense to me and is good enough.

But maybe having a separate, explicit event to dispatch for shell navigation is clearer as you are saying @J-Son89. I'm not sure which is better.